src/plugins/vis_type_timeseries/public/request_handler.ts

```diff
@@ -48,6 +48,8 @@ export const metricsRequestHandler = async ({
       });
 
     try {
+      const searchSessionOptions =
+        searchSessionId && dataSearch.session.getSearchOptions(searchSessionId);
```

Comment:
`getSearchOptions` can return `null` now. TSVB route validation fails in case we send
`searchSession: null`, so this change just makes sure `searchSession` key is not send in case `getSearchOptions` returns null